sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -110,6 +110,7 @@
  */
 public class RxDocumentClientImpl implements AsyncDocumentClient, IAuthorizationTokenProvider, CpuMemoryListener,
     DiagnosticsClientContext {
+    private static final String tempMachineId = "uuid:UUID.randomUUID().toString()";
```

Comment:
why is this initialized to a constant string value of `"uuid:UUID.randomUUID().toString()"` ?